### PR TITLE
ksud: fix issues found by `clippy`

### PR DESCRIPTION
**Pull request by skbeh**
These issues are mostly found by `cargo clippy -- -W clippy::pedantic`.

### CODE REVIEW

File: userspace/ksud/src/event.rs [hunk 1]
```diff
@@ -149,16 +151,14 @@ pub fn on_post_data_fs() -> Result<()> {
...
-    } else {
-        warn!("safe mode, skip module post-fs-data scripts");
     }
 
     Ok(())
 }
 
 pub fn on_services() -> Result<()> {
     // exec modules service.sh scripts
-    if !crate::utils::is_safe_mode() {
```
Comment (tiann) [inline userspace/ksud/src/event.rs:161]: this is wrong

File: userspace/ksud/src/module.rs [hunk 2]
```diff
@@ -197,6 +189,40 @@ pub fn load_sepolicy_rule() -> Result<()> {
...
+    command = command
+        .current_dir(path.as_ref().parent().unwrap())
+        .arg("sh")
+        .arg(path.as_ref())
+        .env("ASH_STANDALONE", "1")
+        .env(
+            "PATH",
+            format!("{}:{}", env_var("PATH").unwrap(), defs::BINARY_DIR),
+        )
+        .env("KSU", "1");
```
Comment (tiann) [inline userspace/ksud/src/module.rs:216]: should be "KSU" "true" instead "1"

**Review by tiann — CHANGES_REQUESTED** [review]

File: userspace/ksud/src/module.rs (hunk 2 above)
Comment (tiann) [reply userspace/ksud/src/module.rs:216]: please modify this, some modules already depend on this behavior

**Review by tiann — APPROVED** [review]

**Comment by skbeh** [comment]
@tiann Is the [cast to `i8`](https://github.com/tiann/KernelSU/blob/219ea1c458fb356b12fc471c1470ffcd2bad29df/userspace/ksud/src/apk_sign.rs#L85) and the [cast to `u32`](https://github.com/tiann/KernelSU/blob/219ea1c458fb356b12fc471c1470ffcd2bad29df/userspace/ksud/src/apk_sign.rs#L91) supposed to overflow?

**Comment by tiann** [comment]
> cast to `i8`

Yes, it is expected behavior

**Pull request merged**